src/app/components/flyout/flyout-guidelines.component.html

```diff
@@ -0,0 +1,273 @@
+<section>
+  <header><div class="sky-section-heading">Usage</div></header>
+  <stache-row>
+    <stache-column screenSmall="6">
+			<p>A flyout is a container that lays over page content to display large amounts of supplementary information related to user tasks. A flyout can contain lists, record views, graphs, or bespoke information displays.</p>
```

Comment:
I didn't address it in my edits, but I don't know what "bespoke information displays" means. Is this right?